src/core/ui/components/drop-down/inner.tsx

```diff
@@ -288,6 +323,7 @@ export const Inner = ({ context }: Props) => {
     authSettings.availableWeb3LoginMethods.length,
     handleConnectClick,
     user,
+    copyCusodialSuccess,
```

Comment:
nit: Let's not use the term `custodial` if we can avoid it. Can we change this to managed?

Reply:
fixed. i'll deploy to beta and test again in local